src/pages/components/alert.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Alert
+description: >
+  An Alert provides urgent contextual feedback in response to user action or system activity.
+package: '@zendeskgarden/react-notifications'
+components:
+  - notifications/src/elements/Alert.tsx
+  - notifications/src/elements/content/Close.tsx
+  - notifications/src/elements/content/Paragraph.tsx
+  - notifications/src/elements/content/Title.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>For urgent feedback</li>
+      <li>For contextual communication inline with page content</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For a passive system update, use a [Notification](/components/notification) instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+By default, an Alert will be read first by a screen reader and is persistent
+until the user dismisses it.
+
+import Default from '../../examples/notifications/alert/AlertDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/notifications/alert/AlertDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Type
+
+Alert types help communicate the tone of the message. They can be Success,
```

Comment:
From a content standpoint, we'd want this statement to be a bit stronger:

`Types indicate what kind of message is in the alert and help the user understand how to respond.`